metabase/metabase
# Add question last cached time info

Epic https://github.com/metabase/metabase/issues/22826

How to test:
- Go to Admin -> Settings -> Caching
- Enable caching for question
- Go to a saved question and open the info panel
- Set caching time to e.g. 1 hour
- Refresh the question
- Refresh the question again after a minute. You should see the time it was cached

<img width="371" alt="Screenshot 2022-07-08 at 13 48 21" src="https://user-images.githubusercontent.com/8542534/177979418-5eb3e050-f13f-4dc1-9857-13b6aa83ce5b.png">
